### PR TITLE
Updated Project documentation and features in Readme

### LINKED ISSUE #423: Improve Project documentation - Readme.md
**Describe the issue**
Now, that all the major features have been done and are satisfying enough at this stage, we can update the documentation with feature updates and instructions. We can keep an under-development section as well for modes like the Meditation whose all screen structures and functioning is ready but just needs to be linked with proper resources.

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by jddeep**
Fixes #423 

**Changes**: Updated Project documentation and features in Readme

**Screenshot/s for the changes**:
> Check updated README.md once merged

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing**: 
> Not available

### CODE REVIEW

**Review by CloudyPadmal — CHANGES_REQUESTED** [review]
Why do we have a 12 mb focus.gif file? Make it smaller or remove. It increases the repo size by 13% just for an image.

File: README.md [hunk 1]
```diff
@@ -21,25 +21,57 @@ A difference to existing projects like OpenBCI is that it will not be necessary
...
-  </tr>
-
-  <tr>
   <td><img src = "/docs/images/Demo.gif" height = "500" width="250"></td>
   <td><img src = "/docs/images/home_screen.png" height = "500" width="250"></td>
-  <td><img src = "/docs/images/nav_menu.png" height = "500" width="250"></td>
+  </tr>
+  
+  <tr>
+  <td><img src = "/docs/images/focus.gif" height = "500" width="250"></td>
```
Comment (CloudyPadmal) [inline README.md]: Link here, resource missing?

File: README.md [hunk 2]
```diff
@@ -21,25 +21,57 @@ A difference to existing projects like OpenBCI is that it will not be necessary
...
+  <td><img src = "/docs/images/settings_screen.jpeg" height = "500" width="250"></td>
   </tr>
 
   </table>
   
+## Features
+
+### Launcher Screen
+
+<td><img src = "/docs/images/home_screen.png" height = "500" width="250"></td>
```
Comment (CloudyPadmal) [inline README.md]: Remove spaces inside tags: src = " to src="

This needs to be fixed in many places

**Review by CloudyPadmal — CHANGES_REQUESTED** [review]
Also get all the images down to 200kb

File: README.md [hunk 3]
```diff
@@ -21,25 +21,57 @@ A difference to existing projects like OpenBCI is that it will not be necessary
...
+
+### Memory graph program mode
+
+<td><img src = "/docs/images/Demo.gif" height = "500" width="250"></td>
+
+This is a data visualization mode with the help of graphs, user-friendly stats and a density-spectrum.
+User can import a dataset into the datalogger from where he can visualize it in the actual mode.
+Implemented with features like play, stop, record, data logger, etc.
+
+### Meditation Program mode(under Development)
```
Comment (CloudyPadmal) [inline README.md]:
```suggestion
### Meditation Program mode (Under Development)
```

File: README.md [hunk 4]
```diff
@@ -21,25 +21,57 @@ A difference to existing projects like OpenBCI is that it will not be necessary
...
 
   </table>
   
+## Features
+
+### Launcher Screen
+
+<td><img src = "/docs/images/home_screen.png" height = "500" width="250"></td>
+
+The four major program modes: Focus, Relax, Memory Graph, Meditation.
```
Comment (CloudyPadmal) [inline README.md]: Put them as a list and set internal hyperlinks

File: README.md [hunk 5]
```diff
@@ -21,25 +21,57 @@ A difference to existing projects like OpenBCI is that it will not be necessary
...
+Each program mode is responsible for specific activities and games with your brain activity.
+
+### Focus Program Mode
+
+<td><img src = "/docs/images/focus_mode.png" height = "500" width="250"></td>
+
+The Focus program mode helps the user to increase focus and concentration power through playing games.
```
Comment (CloudyPadmal) [inline README.md]: Focus program mode helps user increase focus and concentration power by playing games.

File: README.md [hunk 6]
```diff
@@ -21,25 +21,57 @@ A difference to existing projects like OpenBCI is that it will not be necessary
...
+The rocket game comes with features like play, stop, record, program info, datalogger, seek to specific time, etc.
+
+### Memory graph program mode
+
+<td><img src = "/docs/images/Demo.gif" height = "500" width="250"></td>
+
+This is a data visualization mode with the help of graphs, user-friendly stats and a density-spectrum.
```
Comment (CloudyPadmal) [inline README.md]: user-friendly -> user friendly 
density-spectrum -> density spectrum

**Review by CloudyPadmal — CHANGES_REQUESTED** [review]
Lot of content corrections need to be made. Can you check with an English teacher or someone fluent in English to double check the content added

File: README.md (hunk 4 above)
Comment (jddeep) [reply README.md]: @CloudyPadmal Internal hyperlinks as of code of those modes or screenshots?

Comment (CloudyPadmal) [reply README.md]: :Sigh: No. https://github.com/aogilvie/markdownLinkTest/blob/master/README.md

**Review by CloudyPadmal — APPROVED** [review]

**Pull request merged**